evm/common.js

```diff
@@ -0,0 +1,50 @@
+const ethers = require('ethers')
+
+// Setup provider & wallet
+const port = process.env.ETH_HTTP_PORT || `18545`
+const providerURL = process.env.ETH_HTTP_URL || `http://localhost:${port}`
+const privateKey =
+  process.env.PRIVATE_KEY ||
+  '4d6cf3ce1ac71e79aa33cf481dedf2e73acb548b1294a70447c960784302d2fb'
```

Comment:
Can we get a note right here that this private key is for development purposes only, please?

Reply:
Yes! Sorry - had it in there before not sure how it got removed.